smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -84,14 +87,33 @@ public IotedgedLinux(string archivePath, Option<RegistryCredentials> credentials
             this.proxy = proxy;
             this.upstreamProtocol = upstreamProtocol;
             this.requireEdgeInstallation = requireEdgeInstallation;
+            this.overwritePackages = overwritePackages;
         }
 
-        public async Task VerifyNotActive()
+        public async Task VerifyNotInstalled()
```

Comment:
Nit: The name suggests this function verifies, but actually purges in some cases. Rename to 'UpdatePackageState'?

Reply:
Fixed